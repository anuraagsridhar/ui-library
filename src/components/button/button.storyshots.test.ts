
import initStoryshots from '@storybook/addon-storyshots';

const options = {
  configPath: '.storybook',
  storyKindRegex: /^components\.button$/,
};

initStoryshots(options);
