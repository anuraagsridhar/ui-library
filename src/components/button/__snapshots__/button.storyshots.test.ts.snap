// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots components.button Static Examples 1`] = `
<section>
  <fieldset>
    <legend>
      Primary
    </legend>
    <div
      className=""
      onClick={[Function]}
    >
      <div>
        Submit
      </div>
    </div>
  </fieldset>
  <fieldset>
    <legend>
      Secondary
    </legend>
    <div
      className="undefined"
      onClick={[Function]}
    >
      <div>
        Cancel
      </div>
    </div>
  </fieldset>
  <fieldset>
    <legend>
      Loading
    </legend>
    <div
      className=""
      onClick={[Function]}
    >
      <div>
        <i
          className="fas fa-cog fa-spin"
        />
      </div>
    </div>
  </fieldset>
</section>
`;
